index.bs

```diff
@@ -839,6 +850,28 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 1.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
                     bytes of <code>|credentialCreationData|.[=attestationObjectResult=]</code>'s value.
 
+                1.  If <code>|credentialCreationData|.[=AttestationConveyancePreferenceOption=]</code>'s value is
+                    <dl class="switch">
+                        :   "none"
+                        ::  replace potentially uniquely identifying information (such as [=AAGUID=] and
+                            [=attestation certificates=]) in the [=attested credential data=] and [=attestation statement=],
+                            respectively, with blinded versions of the same data.
+
+                        Issue: need to define "blinding".  See issues [#694](https://github.com/w3c/webauthn/issues/694) and
+                            [#462](https://github.com/w3c/webauthn/issues/462).
+
+                        :   "indirect"
+                        ::  The client MAY replace the [=AAGUID=] and [=attestation statement=] with a more privacy-friendly
+                            and/or more easily verifiable version of the same data (for example, by employing a [=Privacy CA=]).
+
+                        :   "direct"
+                        ::  convey the [=authenticator=]'s [=AAGUID=] and [=attestation statement=], unaltered, to the RP.
+
+                        Issue: @balfanz wishes to add to the "direct" case:
+                            If the [=authenticator=] violates the privacy requirements of the [=attestation type=] it is using, 
+                            terminate this algorithm with a "{{AttestationNotPrivateError}}".
```

Comment:
I think the most we can say is that "the UA SHOULD terminate ...", since it takes work to determine that the authentigator is violating the privacy requirements, so UAs won't catch it immediately.

Reply:
ok